### ISSUE TITLE
send upstream PR to rabin-cdc

### ISSUE THREAD

**Issue by max-mapper** [maxogden/rabin#8]
We should contribute the rabin modifications to the upstream source: https://github.com/fd0/rabin-cdc/issues/1

**Comment by green-coder** [comment]
How about first merging #4 into this project **and then** prepare a PR for fd0/rabin-cdc#1 ?

Also, you might want to expose the polynom to the JS API (with get/set modifiers) in case the default doesn't fit some user's use case.

**Comment by max-mapper** [comment]
@green-coder yes that sounds good, I mentioned that here actually https://github.com/fd0/rabin-cdc/issues/1#issuecomment-170279016

**Comment by GamePad64** [comment]
And then you will be able to make src/ a submodule, synchronized with rabin-cdc upstream
